TWTValidation/JSON Validator/Parser/TWTJSONSchemaParser.m

```diff
@@ -85,8 +90,12 @@ - (TWTJSONSchemaTopLevelASTNode *)parseWithError:(NSError *__autoreleasing *)out
         topLevelNode.schema = [self parseSchema:self.JSONSchema];
     } @catch (NSException *exception) {
         topLevelNode = nil;
+        NSError *error = exception.userInfo[TWTJSONExceptionErrorKey];
+        if (!error) {
+            error = [NSError errorWithDomain:TWTJSONSchemaParserErrorDomain code:0 userInfo:@{ NSLocalizedDescriptionKey : @"Uncaught exception" }];
```

Comment:
I added this in case an exception is thrown by the system and not by `failWithErrorCode:...` (which was happening with that property setting bug). Is this reasonable? And if so, what information would be useful?